fetch.js

```diff
@@ -1,7 +1,21 @@
 (function(self) {
   'use strict';
 
-  if (self.fetch) {
+  function canAbortFetch() {
+    if (!self.AbortController || !self.AbortSignal) {
+      return false
+    }
+  
+    var abortController = new self.AbortController()
+  
+    var request = new self.Request('http://a', {
```

Comment:
Presumably, the truthiness of `self.fetch` from L18 indicates that `self.Request` exists. To be more explicit, we could add `self.Request` to L5 if that's preferable.